ci/scrub-kicad-project-files

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+# prototype-kicad-library-ci
+
+# File: /ci/scrub-kicad-project-files
+# Description: prototype-kicad-library-ci repository KiCad project file scrubbing script.
+
+function error()
+{
+    local -r message="$1"
+
+    ( >&2 echo "$mnemonic: $message" )
+}
+
+function abort()
+{
+    if [[ "$#" -gt 0 ]]; then
+        local -r message="$1"
+
+        error "$message, aborting"
+    fi
+
+    exit 1
+}
+
+function display_help_text()
+{
+    echo "NAME"
+    echo "    $mnemonic - Ensure KiCad project file's 'update' and 'last_client' fields are empty."
+    echo "SYNOPSIS"
+    echo "    $mnemonic --help"
+    echo "    $mnemonic --version"
+    echo "OPTIONS"
+    echo "    --help"
+    echo "        Display this help text."
+    echo "    --version"
+    echo "        Display the version of this script."
+    echo "EXAMPLES"
+    echo "    $mnemonic --help"
+    echo "    $mnemonic --version"
+    echo "    $mnemonic"
+}
+
+function display_version()
+{
+    echo "$mnemonic, version $version"
+}
+
+function scrub_kicad_project_files()
+{
+    local kicad_project_files; mapfile -t kicad_project_files < <( git -C "$repository" ls-files '*.pro' | xargs -r -d '\n' -I '{}' find "$repository/{}" ); readonly kicad_project_files
+
+    if ! ( sed -E -e 's/^update=.*$/update=/' -e 's/^last_client=.*$/last_client=/' "${kicad_project_files[@]}" && git -C "$repository" diff --exit-code ); then
```

Comment:
The `sed` invocation suggested in #13 contained an error. By default, `sed` writes the results of its edit to stdout. If the `-i` (`--in-place`) option is used, `sed` will instead edit the files in place. Since the `-i` option is not currently being used, the project files are not edited in place, no diff is generated, and thus the CI job will always pass. Using the `-i` option will fix this.
```suggestion
    if ! ( sed -i -E -e 's/^update=.*$/update=/' -e 's/^last_client=.*$/last_client=/' "${kicad_project_files[@]}" && git -C "$repository" diff --exit-code ); then
```